README.md

```diff
@@ -101,6 +101,11 @@ NOTES:
 1. **YOU MUST READ THE [CONTRIBUTORS GUIDE](CONTRIBUTING.md) BEFORE STARTING TO WORK ON A PULL REQUEST.**
 2. If you have found a vulnerability in the project, please write privately to **security@gitea.io**. Thanks!
 
+## Translating
+
+Translations are done through Crowdin. If you want to translate to a new language ask to add a new language there.
```

Comment:
For new languages you have to contact the maintainer.

You can also just create an issue for language adding or ask on discord on the #translation section

Reply:
So it is not explicit over crowdin.

If you need context or find some translation issues, this is of cours over crowdin. Or via Discord

For general translation questions there is a section in the docs. Currently a bit empty but we hope fo fill it as questions pop up.
https://docs.gitea.io/en-us/translation-guidelines/

Reply:
@Oymate thanks for making a nice sentenc out of my first comment, could you do it for the second one too?